rest-service/manager_rest/test/infrastructure/test_parse_with_resolver.py

```diff
@@ -109,28 +109,25 @@ def mock_create_import_resolver(resolver_section):
         self.assertEqual(create_import_resolver_inputs[0]['rules'],
                          resolver_section['rules'])
 
-        self.assertEqual(self.app.application.parser_context['resolver'],
```

Comment:
this assert checks that after request, the context stays; this is exactly the opposite of what we want to happen